### ISSUE TITLE
Incorrect Java Version

### ISSUE THREAD

**Issue by brandon-gallas** [DIDSR/iMRMC#36]
Dillip, Back in May you solved an iMRMC problem related to the java version of an end user. You appear to have saved your work in the fda archive and it now lives at [archive]\imrmc\branches\CheckVersion

I haven’t seen any merge to the trunk or anything, but I noticed that you made a comment on the releases page, “version number should be at least 1.7 or higher.” When I check the version of my computer, it says, “Version 7 Update 51 (build 1.7.0_51-b13).” Is this synonymous with version 1.7 or should we clarify?

Is the warning on the release page the ultimate solution you recommend or is there something in CheckVersion that could check the version at runtime as part of, or independent of, the iMRMC launch?

Thanks,
Brandon

**Comment by brandon-gallas** [comment]
Hi Dillip,
Just closing out some issues?
Can you let me know ...
Is the warning on the release page the ultimate solution you recommend or is there something in CheckVersion that could check the version at runtime as part of, or independent of, the iMRMC launch?

**Comment by dillipemmanuel** [comment]
Hi Brandon,

Mike and I tried to include some version checking code into iMRMC Java project to check Java version, but it encountered problems because this very same code was blocked from running if the Java version is lower than the compiled version.  So, it is impossible to run just to check the Java version before running the iRMCM application.  We agreed to try Visual C to create CheckVersion application because it will run no matter what Java version is.  But unfortunately, I encountered different problems depending on Windows platform computer especially with the administrative computer.

I realized that a  simple solution for any platform is to get a command prompt and then type java -version, and it will display the version number.  Use this information to determine if iMRMC application can run.  It seems simple instruction will be sufficed.  i.e. Tell users to check Java runtime version of at least 1.7 or higher.

Dillip
